libenv/unix_iface.c

```diff
@@ -180,9 +180,8 @@ static void GetMacAddress(EvalContext *ctx, ARG_UNUSED int fd, struct ifreq *ifr
 
 # elif defined(HAVE_GETIFADDRS) && !defined(__sun)
     char hw_mac[CF_MAXVARSIZE];
-    char *m;
+    char *mac_pointer = NULL;
```

Comment:
Minor note: I'd prefer if you renamed the variable in a separate commit. Diffs would be nicer :)

Reply:
Ok! Fixed. I'll run a new local build since several changes came in with this rebase.